### DISCUSSION TITLE
audio groups lost

### DISCUSSION THREAD

**Discussion by KenBarta** [GrandOrgue/grandorgue#1940] [Help]
Could someone please tell me if audio groups are saved somewhere. I've lost my whole setup. Thought it was saved in "settings" and "combinations" export, but it is not there.

**Comment by oleg68** [comment]
They are saved in the `GrandOrgueConfig` file

**Reply by KenBarta** [reply]
Thanks, but how do you restore them?  I had to re-enter the settings by hand.  I'm using a EMU 1212 sound card, and I have it configured for twelve channels.  It's a large 100 stop organ.

**Comment by oleg68** [comment]
To be precise, tha audio group configuration consits of two parts:
- The list of audio groups and their distribution across audio output channels
    - It is set up in File->Settings->Audio
    - It is stored in the `GrandOrgueConfig` file.
        - One backup is made automatically to the file `GrandOrgueConfig.last` every time you change anything.
        - If is recommended to make the backup of `GrandOrgueConfig` periodically of at least to save screenshots of the File->Settings->Audio tab
    - If you lost this configuration you can restore them either
        - from `GrandOrgueConfig.last`
        - from a manual backup of this file
        - to set it up manually using the screenshot
        - to set it up manually from scratch
- Distribution of organ objects across audio groups
    - It is set up in Audio/Midi -> Organ Settings
    - It is stored in .cmb files
    - It is saved on File->Save or File->Save Settings
    - This settings can be restored
       - from any .cmb file: just Import Settings from it
       - Manually in Audio/Midi -> Organ Settings.
           - For making it easier, keep the audio group settings on the Windchest Level instead of Rank and Pipes because the number of windchests is quite little. 
               - If you are able to change ODF, make separate windchests for mapping to different audio groups. This is an example:
                
![image](https://github.com/GrandOrgue/grandorgue/assets/11586506/d3af1a7f-bbe7-48ab-953e-5a29905de87f)
